test/Quaternion.jl

```diff
@@ -379,10 +379,10 @@ end
             @testset "log" begin
                 @test log(zero(QuaternionF64)) === Quaternion(-Inf, 0, 0, 0)
                 @test log(one(QuaternionF64)) === Quaternion(0.0, 0, 0, 0)
-                @test log(-one(QuaternionF64)) == Quaternion(0.0, π, 0, 0)
+                @test log(-one(QuaternionF64)) ≈ Quaternion(0.0, π, 0, 0)
```

Comment:
Very minor point, but the first argument doesn't have to be `Float64`.
```suggestion
                @test log(-one(QuaternionF64)) ≈ Quaternion(0, π, 0, 0)
```